build/cli.js

```diff
@@ -193,10 +214,25 @@ function humanFileSize(size) {
   return `${num} ${"KMGTPEZY"[i - 1]}B`;
 }
 
-if (require.main === module) {
-  const t0 = new Date();
-  buildDocuments()
-    .then(({ slugPerLocale, peakHeapBytes }) => {
+program
+  .name("build")
+  .option("--spas", "Build the SPA pages", { default: true }) // PR builds
+  .option("--spas-only", "Only build the SPA pages", { default: false })
```

Comment:
The point of these are:

* When you run `yarn prepare-build` you only want to generate the SPAs
* When you run the `pr-build.yml` in `mdn/content` you only care to build the files that changed and you don't want to waste the time building any of these SPA pages. 

If you can think of a better naming scheme or something to put all of this under one option, I'm all ears.